client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/app/Game/Player.java

```diff
@@ -0,0 +1,11 @@
+package edu.colostate.cs.cs414.chesshireCoders.jungleClient.app.Game;
+
+public class Player {
+	PlayerColor colorOfPlayer = PlayerColor.Black;
+	//I'm not sure what this value is supposed to be so I'm changing user to userID
+	int userID = 0;
+
+	public Player() {
```

Comment:
For the Player class, could you add the color and userID as parameters for the constructor and have it set them

Reply:
I have not done player yet, as noted this is only for the board game, the squares, and the pieces. Both the game controller and player are not finished/used.